src/main/java/racingcar/dto/GameResultDto.java

```diff
@@ -0,0 +1,25 @@
+package racingcar.dto;
+
+import java.util.List;
+
+public class GameResultDto {
```

Comment:
응답에는 `Response`가 더 어울릴 것 같아요 :)

```suggestion
public class GameResponse {
```